server/network/aoprotocol.py

```diff
@@ -503,6 +503,9 @@ def net_cmd_ms(self, args):
             if pos not in ('def', 'pro', 'hld', 'hlp', 'jud', 'wit', 'jur',
                            'sea'):
                 return
+        if len(text) > self.server.config['max_chars']:
```

Comment:
This will cause the server to crash if `max_chars` is missing from config.yaml.

Reply:
I forgot to user proof it my bad. Will add it later today